terraform/providers/aws/README.md

```diff
@@ -222,8 +224,10 @@ If you want to create artifacts in other regions, complete these same steps but
   - [ ] From the [root directory](https://github.com/hashicorp/best-practices), navigate to the [`aws_us_east_1_prod` folder](https://github.com/hashicorp/best-practices/blob/master/terraform/providers/aws/aws_us_east_1_prod): `cd terraform/providers/aws/aws_us_east_1_prod/.`
```

Comment:
should it be `cd terraform/providers/aws/us_east_1_prod/` instead? There is no directory aws_us_east_1_prod available.